uk.co.compucorp.civicrm.hrleaveandabsences/views/shared/components/partials/leave-request-dates.html

```diff
@@ -0,0 +1,9 @@
+{{request.from_date | formatDate : null : (request.request_type === 'toil' ? 'hours' : unit)}}
```

Comment:
If request type is `toil`, we are selecting unit as hours. What if the date does not have hours(normal date only), what will be displayed then?

Reply:
In the database we store datetime at the moment. As per PCHR-3427, TOIL will always have times, both "from" and "to". Currently, the datetime is stored as 00:00-23:59 for TOIL. This is why PCHR-3476 exists, it will convert 23:59 to 23:45.

So, answering your question: after PCHR-3427 is finished, it will not be possible to create a TOIL without times specified. Already existing requests will also have times: 00:00-23:45.